app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> None:
+
+    masks_to_buy = 0
+
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
Returning immediately upon encountering a `VaccineError` means the function will stop checking the rest of the friends. If the intention is to check all friends and then decide, consider using a flag or collecting errors and returning a message after the loop.